### PR TITLE
Add Docker configuration files

### PR DESCRIPTION
**Pull request by kj84park**
# What is this PR for?

## Overview or reasons
- Added a Docker configuration and description so that users can easily check the application behavior.

## Tasks
- Add docker execution environment configuration.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      image: kj84park/rp-server:0.0.1
+      ports:
+        - "8080:8080"
+      restart: always
+    fido2:
+      container_name: fido2-server
+      image: kj84park/fido2-server:0.0.1
+      ports:
+        - "8081:8081"
+      restart: always
```
Comment (ric-sapasap) [inline docker-compose.yml:14]: In my opinion, it would be better if the images are built from the local Dockerfiles, instead of pulled from dockerhub.

Comment (ric-sapasap) [reply docker-compose.yml:14]: For the Dockerfiles themselves, we can use multi-stage docker builds so the build process is isolated from the local environment

Comment (kj84park) [reply docker-compose.yml:14]: Thank you so much for the suggestion! :+1:  
Let me check.

Comment (kj84park) [reply docker-compose.yml]: @ric-sapasap  I have changed the settings according to your comments. thank you.
If you have any more comments, please.

Comment (ric-sapasap) [reply docker-compose.yml]: Looks amazing. Thank you very much @kj84park!

**Review by haxelion — APPROVED** [review]
Nothing to add :-)

**Pull request merged**